sys/net/gnrc/transport_layer/tcp/gnrc_tcp_fsm.c

```diff
@@ -174,15 +173,15 @@ static int _transition_to(gnrc_tcp_tcb_t *tcb, fsm_state_t state)
                 return -ENOMEM;
             }
 
-            /* Add to Connections to active connection (if not already active) */
+            /* Add to connection to active connections (if not already active) */
             mutex_lock(&_list_tcb_lock);
             LL_FOREACH(_list_tcb_head, iter) {
                 if (iter == tcb) {
-                    found = true;
+                    break;
                 }
             }
-            /* If not already active: Apped tcb but check portnumber first */
-            if (!found) {
+            /* If not already active: Appned tcb but check portnumber first */
+            if (iter == NULL) {
                 /* Check if Port Number is not in use */
```

Comment:
whats it going to be either _port number_ or (as above) _portnumber_, but please do not mix and no German uppercase either.